lang10/tests/requirements.txt

```diff
@@ -24,8 +24,8 @@ simplejson==3.17.6
 six==1.16.0
 sortedcontainers==2.4.0
 statsd==3.3.0
-symbiont-io.assembly-client~=4.0.0.dev
-symbiont-io.pytest-assembly~=4.0.0.dev
+symbiont-io.assembly-client~=3.1.0.dev6
+symbiont-io.pytest-assembly~=3.1.0.dev6
```

Comment:
Is that correct? Didn't we release 4.0.0?

Reply:
Whoa, I didn't change these on purpose. I'll change this back.